src/main/java/com/minecolonies/coremod/util/TranslationConstants.java

```diff
@@ -0,0 +1,15 @@
+package com.minecolonies.coremod.util;
+
+/**
+ * Constants for translation
+ */
+public final class TranslationConstants
+{
+
+    private TranslationConstants(){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)